src/artemis/devices/oav/oav_centring.py

```diff
@@ -0,0 +1,199 @@
+import json
+import os
+
+import bluesky.plan_stubs as bps
+import bluesky.preprocessors as bpp
+from bluesky import RunEngine
+
+from artemis.devices.oav.oav_detector import OAV, Backlight, Camera, Goniometer
+
+
+class Direction:
+    LEFT_TO_RIGHT = 1
+    RIGHT_TO_LEFT = 2
+    TOP_TO_BOTTOM = 3
+
+
+class NewDirection:
```

Comment:
Nit: I think you should be able to use `IntEnum` (https://docs.python.org/3/library/enum.html#enum.IntEnum) for these